astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -608,6 +608,12 @@ def test_dsplit(self):
 
 
 class TestUfuncReductions(InvariantUnitTestSetup):
```

Comment:
Could you similarly add a test for `np.round` in `TestUfuncLike` (before `test_round_`)?

Sadly, for `test_round_` itself, one will need to filter the deprecation warning...